layouts/_default/single.html

```diff
@@ -6,27 +6,24 @@
   <body class="{{ printf "bg-%s" $bg }}">
     {{ partial "nav.html" . }}
 
-    {{ $page := replace (strings.TrimRight "/" (strings.TrimLeft "/" .RelPermalink)) "/" "__" }}
-
-    {{ with index $.Site.Data $page }}
-      {{ range . }}
-        {{ $.Scratch.Set "data" . }}
-
-        {{ if and (not .disabled) (eq .fragment "hero") }}
-          {{ partial "fragments/hero.html" $ }}
-        {{ end }}
+    {{ range sort .Resources }}
 
+      {{ if and (not .Params.disabled) (eq .Params.fragment "hero") }}
+        {{ partial "fragments/hero.html" .Params }}
       {{ end }}
+
     {{ end }}
 
-    {{ partial "content-single.html" . }}
+    {{ $page := replace (strings.TrimRight "/" (strings.TrimLeft "/" .RelPermalink)) "/" "__" }}
+
+     {{ partial "content-single.html" . }}
 
     {{ with index $.Site.Data $page }}
       {{ range sort . "weight" }}
         {{ $.Scratch.Set "data" . }}
-
-        {{ if and (not .disabled) (eq .fragment "items") }}
-          {{ partial "fragments/items.html" $ }}
+      
```

Comment:
Whitespace is unnecessary here. There might be an option in your editor to highlight or auto delete such whitespace.

Reply:
Whitespace is still there. To be clear the line has "       " whitespace/spaces included.